### PR TITLE
Update to pac4j v1.7

### PR DESCRIPTION
**Pull request by leleuj**

### CODE REVIEW

File: src/main/java/org/pac4j/undertow/DemoHandlers.java [hunk 1]
```diff
@@ -138,8 +146,24 @@ public void handleRequest(HttpServerExchange exchange) throws Exception {
                 sb.append("<input type=\"submit\" name=\"submit\" value=\"Submit\" />");
                 sb.append("</form>");
 
-                exchange.getResponseSender().send(sb.toString());
+                returnHtml(exchange, sb.toString());
             }
         };
     }
+
```
Comment (miremond) [inline src/main/java/org/pac4j/undertow/DemoHandlers.java]: I agree, it seems complicated. What is your use case exactly here?

Comment (leleuj) [inline src/main/java/org/pac4j/undertow/DemoHandlers.java]: In fact, it doesn't even work: I just kept the source code to remember.

When trying to access a protected area, if you give up during the authentication process (like for Twitter), a 403 page is returned: it's a blank page here and I'd like to set my own 403 page...

Comment (miremond) [inline src/main/java/org/pac4j/undertow/DemoHandlers.java]: I am not sure but is it not implemented here:
https://github.com/pac4j/undertow-pac4j/blob/master/src/main/java/org/pac4j/undertow/ClientAuthenticationMechanism.java#L78
?


2015-04-30 8:45 GMT+02:00 LELEU Jérôme <notifications@github.com>:

Comment (leleuj) [inline src/main/java/org/pac4j/undertow/DemoHandlers.java]: In fact, the problem is to catch the 403 error page to display our own page...

**Pull request closed without merging**